### PR TITLE
Replace cxxfilt with itanium_demangler

### PR DESCRIPTION
**Pull request by cHemingway**
Removes need for libc, allowing metal-serial to run on windows
Closes issue #3

### CODE REVIEW

**Comment by cHemingway** [comment]
Failed to build as itanium_demangler requires setuptools, which isn't installed yet. Think this is fixable by putting setuptools first in requirements.txt, will try.

**Comment by cHemingway** [comment]
Well that didn't work either, beats me!

**Comment by klemens-morgenstern** [comment]
Thanks I will check.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 argparse~=1.4.0
 pcpp~=1.21
 pyelftools~=0.26
-cxxfilt~=0.2.1
+itanium_demangler~=1.0
```
Comment (klemens-morgenstern) [inline requirements.txt:5]:
```suggestion
setuptools~=39.0.1
itanium_demangler~=1.0
```

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 twine~=3.1.1
 setuptools~=39.0.1
```
Comment (klemens-morgenstern) [inline requirements.txt]:
```suggestion
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -20,6 +20,9 @@ jobs:
     steps:
     - name: Install gdb
       run: sudo apt-get install gdb
+      # Install python-setuptools system wide
+    - name: Install python-setuptools
+      run: sudo apt-get install python-setuptools
```
Comment (klemens-morgenstern) [inline .github/workflows/main.yml]:
```suggestion
      run: sudo apt-get install python3-setuptools
```

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 argparse~=1.4.0
 pcpp~=1.21
 pyelftools~=0.26
-cxxfilt~=0.2.1
+setuptools~=39.0.1
+itanium_demangler~=1.0
 twine~=3.1.1
-setuptools~=39.0.1
\ No newline at end of file
+setuptools~=39.0.1
```
Comment (klemens-morgenstern) [inline requirements.txt]: Just tidying up
```suggestion
```

**Comment by cHemingway** [comment]
Thanks for fixing that, I missed the typo in python**3**-setuptools.
Are you happy to merge, or do you want any changes?

**Comment by klemens-morgenstern** [comment]
Looks, good to me, waited for the test to finish & forgot to hit merge ;).

**Comment by klemens-morgenstern** [comment]
Should be release on pip shortly with https://github.com/metal-ci/test/releases/tag/v0.2.3

**Pull request merged**